lms/startup.py

```diff
@@ -41,7 +43,7 @@ def run():
         enable_stanford_theme()
 
     if settings.FEATURES.get('USE_MICROSITES', False):
```

Comment:
We should update this to use microsite.is_feature_enabled(), after it becomes available (see above comment)